dev-packages/application-package/package.json

```diff
@@ -35,23 +35,12 @@
     "@types/write-json-file": "^2.2.1",
     "bunyan": "^1.8.10",
     "changes-stream": "^2.2.0",
-    "circular-dependency-plugin": "^2.0.0",
-    "copy-webpack-plugin": "^4.0.1",
-    "css-loader": "^0.28.1",
     "electron": "1.7.11",
```

Comment:
should go the application manager

Reply:
Oops I had those changes, I just forgot to commit them!

Reply:
Please move `electron`, `less`, `font-awesome-webpack` dependencies to the manager.